signals.mss

```diff
@@ -795,6 +795,25 @@ Format details:
     }
   }
 
+  /****************************/
+  /* NL distant light signals */
+  /****************************/
+  ["feature"="NL"]["distant_form"="light"] {
```

Comment:
This looks a bit strange to me, all other signals have ```COUNTRY:signal_name```, while this one only has ``NL``.

Reply:
Yeah I agree, but that is the current tagging scheme, as documented in https://wiki.openstreetmap.org/wiki/OpenRailwayMap/Tagging_in_Netherlands

The numbering as shown in the signal book might also be used, but at the moment things are not tagged, nor documented this way. https://www.dutchrail.eu/spoorwegen/nederland/seinenboek?showall=1

The scope of mass-retagging would be quite large, I'm not sure if that is an option.

Reply:
I will add support in this PR to ensure the current tagging (`NL` as feature name) is allowed, and also the specific feature name like `NL:201` for a main signal.

Reply:
I added alternative tagging that can be migrated towards, with a name per main feature (`NL:hoofd`, `NL:voor` and `NL:herhaling`). The speed signal is part of the main signal  `NL:hoofd` feature.